docs/src/capi/tutorial/getting-started.md

```diff
@@ -1 +1,196 @@
+
+The `clusterctl` CLI tool manages the lifecycle of a Cluster API management
+cluster. To install it, follow the [upstream instructions]. Typically, this
+involves fetching the executable that matches your hardware architecture and
+placing it in your PATH. For example, at the time this guide was written,
+for `amd64` you would run:
+
+```sh
+curl -L https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.7.3/clusterctl-linux-amd64 -o clusterctl
+sudo install -o root -g root -m 0755 clusterctl /usr/local/bin/clusterctl
+```
+
+### Configure clusterctl
+
+`clusterctl` contains a list of default providers. Right now, Canonical
+Kubernetes is not yet part of that list. To make `clusterctl` aware of the
+Canonical K8s providers, we need to add a clusterctl configuration file.
+
+```sh
+mkdir -p ~/.config/cluster-api
+curl -L https://raw.githubusercontent.com/canonical/cluster-api-k8s/main/clusterctl.yaml -o ~/.config/cluster-api/clusterctl.yaml
+```
+
+### Set up a management cluster
+
+The management cluster hosts the CAPI providers. You can use Canonical
+Kubernetes as a management cluster:
+
+```sh
+sudo snap install k8s --classic --edge
+sudo k8s bootstrap
+sudo k8s status --wait-ready
+mkdir -p ~/.kube/
+sudo k8s config > ~/.kube/config
+```
+
+When setting up the management cluster, place its kubeconfig under
+`~/.kube/config` so other tools such as `clusterctl` can discover and interact
+with it.
+
+### Prepare the infrastructure provider
+
+Before generating a cluster, you need to configure the infrastructure provider.
+Each provider has its own prerequisites. Please follow the instructions
+for your provider:
+
+````{tabs}
+```{group-tab} AWS
+
+The AWS infrastructure provider requires the `clusterawsadm` tool to be
+installed:
+
+```sh
+curl -L https://github.com/kubernetes-sigs/cluster-api-provider-aws/releases/download/v2.5.2/clusterawsadm-linux-amd64 -o clusterawsadm
+chmod +x clusterawsadm
+sudo mv clusterawsadm /usr/local/bin
+```
+
+`clusterawsadm` helps you bootstrapping the AWS environment that CAPI will use
+It will also create the necessary IAM roles for you.
+
+Start by setting up environment variables defining the AWS account to use, if
+these are not already defined:
+
+```sh
+export AWS_REGION=<your-region-eg-us-east-1>
+export AWS_ACCESS_KEY_ID=<your-access-key>
+export AWS_SECRET_ACCESS_KEY=<your-secret-access-key>
+```
+
+If you are using multi-factor authentication, you will also need:
+
+```sh
+export AWS_SESSION_TOKEN=<session-token>
+```
+
+`clusterawsadm` uses these details to create a [CloudFormation] stack in your
+AWS account with the correct [IAM] resources:
+
+```sh
+clusterawsadm bootstrap iam create-cloudformation-stack
+```
+
+The credentials need to be encoded and stored as a Kubernetes secret:
+
+```sh
+export AWS_B64ENCODED_CREDENTIALS=$(clusterawsadm bootstrap credentials encode-as-profile)
+```
+
+You are now all set to deploy the AWS CAPI infrastructure provider.
+
+```
```

Comment:
Something funny is happening here, making this an empty code block. See https://canonical-ubuntu-documentation-library--540.com.readthedocs.build/canonical-kubernetes/540/capi/tutorial/getting-started/.
Maybe @evilnick knows why?

Reply:
From a quick glance, you need to balance and add backticks to make it render properly. So if you have triple-tick code blocks, the 'parent' of that needs four ticks and so on.

Reply:
and remember to close things too

Reply:
thanks, that worked